app/index.css

```diff
@@ -15,22 +18,51 @@ body {
   justify-content: center;
   width: 100%;
   height: 100%;
-  background: #eee;
+  background: $gray;
   & > p {
     font-size: 3rem;
   }
 }
 
-h1 {
-  margin: 0;
-  padding-left: 1rem;
-  padding-top: 1rem;
+$drawer-width: 20vw;
+
+.plugins,
+.container {
+  position: absolute;
+  top: 0;
+  height: 100vh;
 }
 
-pre {
-  padding: 1rem;
-  background: #eee;
-  border-radius: 0.3rem;
+.plugins {
+  left: -$drawer-width;
+  width: $drawer-width;
+  background: $dark;
+  overflow-y: scroll;
```

Comment:
I have a scrolling issue where sidebar is cut at the bottom. I think we should set `overflow: auto;` for both .container and .plugins